extensions/hacker-news/src/frontpage.tsx

```diff
@@ -75,6 +142,31 @@ function Actions(props: { item: Parser.Item }) {
           />
         )}
       </ActionPanel.Section>
+
+      <ActionPanel.Section>
+        <Action
+          title="Toggle Summarize Window"
+          shortcut={{ modifiers: ["cmd"], key: "]" }}
+          onAction={async () => {
+            if (openAI != null) props.setIsShowingDetail((prev) => !prev);
```

Comment:
There is no check for Raycast AI here, also, is there a way to just show the result in the next screen without having to run another action panel item?

Reply:
Hi, in line 22 `const openAI = new OpenAIModule(openAIKey);` actually, it shouldn't be null because it initializes it. In OpenAIModule, there is a logic that will check if you are using RaycastAI or not.
In OpenAIModule class you can see
`if (environment.canAccess(AI)) return await this.raycastAiRequest(prompt);
    else if (this.openai != null) return await this.gptRequest(prompt);
    else throw new Error("AI module not initialized");`

Reply:
Actually, it was tough because there should be some action that will toggle that window. One action can toggle the window and call AI, but if you go to the next item, it will automatically make an AI request that costs money. So, in this way, I'm making sure that the user actually wants to call AI to summarize the article.